packages/components/src/templates/carousel-with-images.stories.ts

```diff
@@ -0,0 +1,50 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Carousel with Images',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: ''
+    }
+  }
+};
+
+export const Default = {
+  name: 'Default',
+  render: () => html`
+    <sd-carousel>
```

Comment:
Instead of generic placeholder ... show different placeholder images from the placeholder package, pls.

Reply:
Updated

Reply:
Updated